projects/controllers/components.py

```diff
@@ -134,6 +136,16 @@ def get_component(uuid):
     return component.as_dict()
 
 
+def get_components_by_tag(tag):
+    """Get all components with a specific tag.
```

Comment:
Pode criar um unit test para esta função?
Por que precisa dessa função? Não vi uma API que a usa.

Reply:
To usando pra pega o componente com a tag dataset pra cria um experimento com ele